src/braket/circuits/gates.py

```diff
@@ -457,10 +456,10 @@ class Rx(AngledGate):
     """X-axis rotation gate.
 
     Args:
-        angle (Union[FreeParameter, float]): angle in radians.
+        angle (Union[FreeParameterExpression, float]): angle in radians.
```

Comment:
These changes seem unrelated to noise. Why weren't they done when FreeParameterExpression was created?

Reply:
Just happened to be missed at the time